src/main/java/com/capgemini/fs/coindashboard/encryptionService/AESService.java

```diff
@@ -0,0 +1,36 @@
+package com.capgemini.fs.coindashboard.encryptionService;
+
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+import javax.crypto.*;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+public class AESService {
```

Comment:
if we keep secrets in external file on a host, why do we need that encrypting service ?

Reply:
That's a valid point. [Alternative](https://github.com/CuttingEdgeCommunity/coindashboard-backend/pull/35) uses that approach to avoid over-complicating. This is a good reason for rejecting this pull request.